### ISSUE TITLE
Implement Object Select feature

### ISSUE THREAD

**Issue by Evan-CTL** [ccnmtl/mathplayground#205]
Implement an object select to specify which object should be affected by hotkey manipulation, including shifting selection up and down the object list with the keyboard for accessibility purposes. As it stands using a hotkey will affect all objects in the scene at once.
